### ISSUE TITLE
Rare NPC - Ranger Lord Hawkspear - Neutral and attackable by Alliance

### ISSUE THREAD

**Issue by IntelligentQuantum** [azerothcore/azerothcore-wotlk#8324]
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [ ] Horde

### Content Phase:

- [X] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [X] 50-59

### Current Behaviour

Original Post: https://github.com/chromiecraft/chromiecraft/issues/1997

The rare NPC [Ranger Lord Hawkspear](https://wowgaming.altervista.org/aowow/?npc=10824) in Eastern Plaguelands appears as neutral and thus attackable for Alliance players.

[Picture of me killing him.](https://pasteboard.co/jTLSxBcQVGjT.jpg)

### Expected Blizzlike Behaviour

Friendly to Alliance
Hostile to Horde

### Source

https://classic.wowhead.com/npc=10824/ranger-lord-hawkspear

### Steps to reproduce the problem

Check the NPC as Alliance.

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/b2f08934bcc60491f8fa8971f98837b2d48a489e

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by gurveleg** [comment]
Indeed, the change to neutral to both sides came in Cataclysm when he turned into https://wowpedia.fandom.com/wiki/Death-Hunter_Hawkspear
